content/en/docs/reference/instrumentation/metrics.md

```diff
@@ -0,0 +1,325 @@
+---
+title: Kubernetes Metrics Across Components
+content_type: instrumentation
+---
+
+
+## Metrics
+
+These are the metrics which are exported in Kubernetes components (i.e. kube-apiserver, scheduler, kube-controller-manager, kube-proxy, cloud-controller-manager). 
+
+(auto-generated 2022 Oct 25)
+
+### List of Kubernetes Metrics
+
+<table class="table">
+<thead>
+    <tr>
+        <td width="20%">Name</td>
+        <td width="10%">Stability Level</td>
+        <td width="10%">Type</td>
+        <td width="30%">Help</td>
+        <td width="20%">Labels</td>
+        <td width="10%">Const Labels</td>
+    </tr>
+</thead>
+<tbody>
+<tr><td>etcd_version_info</td><td>ALPHA</td><td>Gauge</td><td>Etcd server's binary version</td><td><div>binary_version</div></td><td>None</td></tr>
```

Comment:
It'd be handy to set a class on the different `<td>` elements in each row, again to allow custom styling.

Reply:
What class do you recommend?

Reply:
An example:
```html
<tr>
  <td class="metric_name">etcd_version_info</td>
  <td class="metric_stability_level" data-stability="alpha">Alpha</td> <!-- use CSS to capitalize if preferred -->
  <td class="metric_time_series_type" data-type="gauge">Gauge</td>
  <td class="metric_description">etcd server's binary version</td>
  <td class="metric_labels_varying"><span class="metric_label">binary_version<span></td>
  <td class="metric_labels_constant">None</td>
</tr>
```

This leaves room for CSS and JavaScript to be added later. It's nice to get it in early and automatically, so that localization teams can minimize their effort when we add features.